### ISSUE TITLE
ERROR: Failed to generate bibliography in Quarto 99.9.9

### ISSUE THREAD

**Issue by camilogarciabotero** [quarto-dev/quarto-cli#2851]
### Bug description

Hi,

I'm re-render a quarto [book](https://github.com/camilogarciabotero/biocomp-book) with the latest version (99.99) commit 8c2b10dc4. 

And got the following error: 

```

ERROR: Failed to generate bibliography, rendering failed with code 1
pandoc: /Users/camilogarcia/Documents/Teaching/BiologyCourses/BI0487/Lectures/biocomp-book/Users/camilogarcia/Documents/Teaching/BiologyCourses/BI0487/Lectures/biocomp-book/references.bib: openBinaryFile: does not exist (No such file or directory)


Error: Failed to generate bibliography, rendering failed with code 1
pandoc: /Users/camilogarcia/Documents/Teaching/BiologyCourses/BI0487/Lectures/biocomp-book/Users/camilogarcia/Documents/Teaching/BiologyCourses/BI0487/Lectures/biocomp-book/references.bib: openBinaryFile: does not exist (No such file or directory)

    at renderToCSLJSON (file:///Users/camilogarcia/Applications/quarto-cli/src/core/bibliography.ts:99:13)
    at async bibliographyCslJson (file:///Users/camilogarcia/Applications/quarto-cli/src/core/bibliography.ts:22:24)
    at async googleScholarReferences (file:///Users/camilogarcia/Applications/quarto-cli/src/format/html/format-html-meta.ts:241:22)
    at async file:///Users/camilogarcia/Applications/quarto-cli/src/format/html/format-html-meta.ts:42:33
    at async file:///Users/camilogarcia/Applications/quarto-cli/src/command/render/render.ts:392:24
    at async withTimingAsync (file:///Users/camilogarcia/Applications/quarto-cli/src/core/timing.ts:50:20)
    at async runHtmlPostprocessors (file:///Users/camilogarcia/Applications/quarto-cli/src/command/render/render.ts:383:5)
    at async renderPandoc (file:///Users/camilogarcia/Applications/quarto-cli/src/command/render/render.ts:169:33)
    at async Object.onRender (file:///Users/camilogarcia/Applications/quarto-cli/src/project/types/book/book-render.ts:214:28)
    at async renderFiles (file:///Users/camilogarcia/Applications/quarto-cli/src/command/render/render-files.ts:482:11)
```

I'm on macOS:

```
                    'c.          
                 ,xNMM.          ----------------------
               .OMMMMo           OS: macOS 12.6 21G115 x86_64
               OMMM0,            
     .;loddo:' loolloddol;.      Kernel: 21.6.0
   cKMMMMMMMMMMNWMMMMMMMMMM0:    Uptime: 1 day, 23 hours, 46 mins
 .KMMMMMMMMMMMMMMMMMMMMMMMWd.    Packages: 211 (brew)
 XMMMMMMMMMMMMMMMMMMMMMMMX.      Shell: zsh 5.8.1
;MMMMMMMMMMMMMMMMMMMMMMMM:       Resolution: 1440x900, 1920x1080
:MMMMMMMMMMMMMMMMMMMMMMMM:       DE: Aqua
.MMMMMMMMMMMMMMMMMMMMMMMMX.      WM: Rectangle
 kMMMMMMMMMMMMMMMMMMMMMMMMWd.    Terminal: iTerm2
 .XMMMMMMMMMMMMMMMMMMMMMMMMMMk   Terminal Font: JetBrainsMono-Regular 18
  .XMMMMMMMMMMMMMMMMMMMMMMMMK.   CPU: Intel i5-8257U (8) @ 1.40GHz
    kMMMMMMMMMMMMMMMMMMMMMMd     GPU: Intel Iris Plus Graphics 645
     ;KMMMMMMMWXXWMMMMMMMk.      Memory: 6236MiB / 8192MiB
       .cooc,.    .,coo:.
```

It looks like the `references.bib` file path is being wrongly generated in the function `renderToCSLJSON ` probably in the `cmd.push(join(Deno.cwd(),join(dir,biblio)));` line.

```
export async function renderToCSLJSON(
  dir: string,
  biblios: string[],
): Promise<CSL[]> {
  const bibloEntries = [];
  for (const biblio of biblios) {
    const cmd = [pandocBinaryPath()];
    cmd.push(join(Deno.cwd(), join(dir, biblio)));
    cmd.push("-t");
    cmd.push("csljson");
    cmd.push("--citeproc");

    const result = await execProcess(
      { cmd, stdout: "piped", stderr: "piped", cwd: dir },
    );
    if (result.success) {
      if (result.stdout) {
        const entries = JSON.parse(result.stdout);
        bibloEntries.push(...entries);
      }
    } else {
      throw new Error(
        `Failed to generate bibliography, rendering failed with code ${result.code}\n${result.stderr}`,
      );
    }
  }
  return bibloEntries;
}
```

Thanks!

### Checklist

- [X] Please include a minimal, fully reproducible example in a single .qmd file? Please provide the whole file rather than the snippet you believe is causing the issue.
- [X] Please [format your issue](https://quarto.org/bug-reports.html#formatting-make-githubs-markdown-work-for-us) so it is easier for us to read the bug report.
- [X] Please document the RStudio IDE version you're running (if applicable), by providing the value displayed in the "About RStudio" main menu dialog?
- [X] Please document the operating system you're running. If on Linux, please provide the specific distribution.

**Comment by dragonstyle** [comment]
Is the CSL path that you're providing in the causal document an absolute path?

**Comment by camilogarciabotero** [comment]
Well, I'm not providing any specific CSL file, just running on the default settings for the style.

**Comment by camilogarciabotero** [comment]
No, this is my `_quarto.yml`:

```
project:
  type: book
  output-dir: docs

book:
  title: "Fundamentals of computational biology"
  subtitle: "Lecture notes"
  author: 
    - name: "Camilo García-Botero"
      orcid: 0000-0002-0426-7007
      url: https://camilogarciabotero.github.io/
      affiliation: Universidad EAFIT
  cover-image: figs/cover-03.png
  # search: true  

  # citation:
  #   type: book
  #   # container-title: "Journal of Data Science Software"
  #   doi: 10.5281/zenodo.6468280
  #   url: https://camilogarciabotero.github.io/biocomp-book/

  date: today
  date-format: "YYYY-MM-DD"

  repo-url: https://github.com/camilogarciabotero/biocomp-book
  site-url: https://camilogarciabotero.github.io/biocomp-book/
  repo-actions: [edit, issue]
  repo-branch: main
  # downloads: [pdf]
  google-analytics: "G-QFWWJGZHLD"
  # sharing: [twitter]
  sidebar:
    style: floating

  page-footer: 
    border: true
    center: "Made with [Quarto](https://quarto.org/). Under [GNU GPL 3.0](https://github.com/camilogarciabotero/biocomp-book/blob/main/LICENSE). Buy me coffee [{{< fa mug-saucer >}}](https://www.paypal.com/paypalme/camilogarciabotero) and visit me at [{{< fa house >}}](https://camilogarciabotero.github.io/)"

  chapters:
    - index.qmd
    - intro.qmd
    - part: "The command line"
      chapters:
        - chs-command-line/welcome-to-command-line.qmd
        - chs-command-line/git-and-github.qmd
        - chs-command-line/package-managers.qmd
        - chs-command-line/hpc-notions.qmd
    - part: "Sequence analysis"
      chapters:
        - chs-sequence-analysis/introduction-to-sequence-analysis.qmd
        - chs-sequence-analysis/sanger-analysis.qmd
        - chs-sequence-analysis/sequence-alignments.qmd
        - chs-sequence-analysis/NGS-TGS-principles.qmd
    - part: "Phylogenetics"
      chapters:
        - chs-phylogenetics/phylogenetic-reconstructions.qmd
        - chs-phylogenetics/phylogenomics.qmd
    - part: "Genomics"
      chapters:
        - chs-genomics/genome-assembly.qmd
        - chs-genomics/genome-annotation.qmd
        - chs-genomics/variant-calling-analysis.qmd
        - chs-genomics/comparative-genomics.qmd
    - part: "Metagenomics"
      chapters:  
        - chs-metagenomics/metagenomic-sequencing.qmd
    - part: "Structural bioinformatics"
      chapters:
        - chs-bio-structural/intro-structural-bioinfo.qmd
        - chs-bio-structural/molecular-docking.qmd
    - part: "Challenges"
      chapters:
        - chs-challenges-demos/unix-challenges.qmd
        - chs-challenges-demos/seq-analysis-challenges.qmd
    #     - chs-challenges-demos/demo-sanger-processing.qmd
    #     - chs-challenges-demos/demo-sequence-alignment.qmd
    #     - chs-challenges-demos/demo-phylogenetics.qmd
    #     - chs-challenges-demos/demo-reads-assessment.qmd
    #     - chs-challenges-demos/demo-genome-assembly.qmd
    - part: Appendix
      chapters: 
        - HISTORY.qmd 
    - references.qmd

bibliography: references.bib
reference-location: margin
crossref: 
  lst-title: Code

comments:
  hypothesis: true
  giscus: 
      repo: camilogarciabotero/biocomp-book

execute:
  cache: true

google-scholar: true

format:
  html:
    theme: 
      light: 
      - flatly
      - style/fonts.scss
      dark: 
      - darkly
      - style/fonts.scss
    code-link: true
    code-copy: hover
    mainfont: Atkinson Hyperlegible
    monofont: JetBrains
    highlight-style: ayu
    # mainfont: sora Fira Code nord github oblivion arrow
  # pdf:
  #   documentclass: book
  #   code-block-border-left: false
  #   toc-depth: 2
  #   lof: true
  #   lot: true
  #   title-bg-image: figs/cover.png
  # pdf:
  #   documentclass: scrreprt
```

**Comment by dragonstyle** [comment]
This should be fixed as of v1.2.215 - thank you!

**Comment by camilogarciabotero** [comment]
Wow! That was quite a fast fix. Thank you!
